src/simplicial-hott/05-segal-types.rzk.md

```diff
@@ -397,8 +399,10 @@ functions or extensions into a family of Segal types is again a Segal type.
             (\{t : 2 * 2 | BOT} -> recBOT)))
 ```
 
-```rzk title="RS17, Corollary 5.6(ii)" : if X is a shape and A : X -> U is such that
--- A(x) is a Segal type for all x then (x : X) -> A x is a Segal type
+If X is a shape and A : X -> U is such that A(x) is a Segal type for all x then
+(x : X) -> A x is a Segal type.
```

Comment:
Same here. If you prefer, you can also highlight it as code using single backticks.

Reply:
An argument for highlighting as code would be instances where we use unicode characters. See for instance the comment:

`The Segal-associativity-witness curries to define a diagram Δ²×Δ¹ -> A. The Segal-associativity-tetrahedron is extracted via the middle-simplex map `\ ((t , s) , r) -> ((t , r) , s)` from Δ³ to Δ²×Δ¹.`

Should we use `Segal-associativity-witness` to reference named terms? And what about the shapes `Δ³` and `Δ²×Δ¹`?

Reply:
I don't understand your agument against Unicode in latex. Latex should handle most unicode well. On the other hand, writing the name of functions like `Segal-associativity-witness` in latex is kind of awkward. Maybe it is best to always highlight as code, although I do think there are instances where latex looks better.

Reply:
I it iwould be better to have code highlighting as the _default_ at least.

Reply:
Just pushed an update which uses LaTeX for the markdown math except when it names a term, in which case that is done with code highlighting. Let me know what you think.